pkg/common/utils/k8s/client.go

```diff
@@ -229,6 +229,20 @@ func GetConfigMap(ctx context.Context, k8scient client.Client, namespace, name s
 	return &configMap, nil
 }
 
+// CheckSecretExist check if the secret is in the namespace.
```

Comment:
ChecSecretExist, check the secret exist or not in specify namespace.

Reply:
Done, and I moved this method to sub_controller.go